core/common/src/main/java/alluxio/retry/ExponentialBackoffRetry.java

```diff
@@ -52,7 +51,9 @@ protected long getSleepTime() {
       // current logic overflows at 30, so set value to max
       return mMaxSleepMs;
     } else {
-      int sleepMs = mBaseSleepTimeMs * Math.max(1, mRandom.nextInt(1 << (count + 1)));
+      // use randomness to avoid contention between many operations using the same retry policy
+      int sleepMs =
+          mBaseSleepTimeMs * (ThreadLocalRandom.current().nextInt(1 << count, 1 << (count + 1)));
```

Comment:
It's javadoc says it's well-suited for use by concurrent programs, why do you think it should not be used concurrently

> When applicable, use of ThreadLocalRandom rather than shared Random objects in concurrent programs will typically encounter much less overhead and contention
https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ThreadLocalRandom.html